### DISCUSSION TITLE
Text generation does not work on iPad

### DISCUSSION THREAD

**Discussion by bluabaleno** [nhaouari/obsidian-textgenerator-plugin#56] [General]
Hello,

Whenever I try to use it, the app seems to freeze and does not produce any text.

![image](https://user-images.githubusercontent.com/14120416/207728758-ebc07bf9-78e2-45ca-abe7-a070bacbfcd5.png)

Is there is anything that can be done to fix this issue? appreciate it!

**Comment by dionvansevenant** [comment]
Just encountered this, myself.

Check your OpenAI account for usage credits.

Unfortunately the API doesn't seem to offer a way for the author of the plugin to check usage credits before trying to process text, and the error message may or may not be explicitly related to usage credits.

However, it does seem to correlate.

-- Dion
